client/src/components/APIDataTypes.ts

```diff
@@ -80,9 +80,29 @@ export type AddressRecord = {
   zip: string | null;
 };
 
+export type PortfolioGraphNode = {
+  id: number;
+  value: {
+    kind: "name" | "bizaddr";
+    value: string;
+  };
+};
+
+export type PortfolioGraphEdge = {
+  from: number;
+  to: number;
+  reg_contacts: number;
+};
+
+export type RawPortfolioJson = {
+  nodes: PortfolioGraphNode[];
+  edges: PortfolioGraphEdge[];
+};
```

Comment:
I wonder if this should be called `RawPortfolioGraphJson`, just so it's not confused with the legacy JSON structure that describes portfolios?

Also, I updated this typing in 2cabd8d7b817f238da8586c977c0e3af977afa24 so it doesn't have any `any`'s anymore.

Reply:
noice!